### PR TITLE
Support filtering provisioned services on overview

### PR DESCRIPTION
**Pull request by spadgett**
screen shot 2017-04-19 at 8 28 05 am

### CODE REVIEW

File: app/scripts/controllers/newOverview.js [hunk 1]
```diff
@@ -301,11 +301,15 @@ function OverviewController($scope,
   };
 
   var filterByLabel = function(items) {
-    return LabelFilter.getLabelSelector().select(items);
+    var labelSelector = LabelFilter.getLabelSelector();
+    if (labelSelector.isEmpty()) {
+      return items;
+    }
```
Comment (spadgett) [inline app/scripts/controllers/newOverview.js]: @jwforres The label `LabelSelector.select()` rejects objects with no labels even when there are no active filters. Is this a label selector bug?

**Comment by jwforres** [comment]
Not a bug, it's an option when you instantiate the LabelSelector. There are
times when we want empty to select all (when being used as a filter) and
times where we want empty to select nothing. Conceptually this would be
like a Service with no selector set wouldn't select any Pods.

**Comment by spadgett** [comment]
Requires https://github.com/kubernetes-ui/label-selector/pull/31

**Comment by spadgett** [comment]
@jwforres updated

**Comment by jwforres** [comment]
[merge]

**Pull request merged**